src/GUI/efpt30.core/EFCoreMigrationsBuilder.cs

```diff
@@ -78,16 +78,8 @@ public class EfCoreMigrationsBuilder
 
             foreach (var type in types)
             {
-                try
-                {
-                    var dbContext = operations.CreateContext(type.Name);
-                    result.Add(new Tuple<string, string>(type.Name, GetMigrationStatus(dbContext)));
-                }
-                catch (InvalidOperationException ex)
```

Comment:
Keep this in, it is in order to skip DbContext that cannot be created, and continue

Reply:
If the exception is handled and the user cannot know about it, why not tell the user?